README.md

```diff
@@ -1,10 +1,30 @@
-# interop-2023-mobile-testing
-Mobile Testing Investigation in Interop 2023
+# interop-mobile-testing
+Mobile Testing Investigation in Interop
 
-## Scope
+## 2024 Investigation
+
+### Scope
+
+Do the work so that by Interop 2025 the dashboard has a mobile view, and the mobile view is considered as reliable as desktop.
+
+### Roadmap
+
+1. Formulate and implement a plan for getting Mobile Safari Results onto the dashboard
```

Comment:
```suggestion
1. Formulate and implement a plan for getting iOS WebKit Results onto the dashboard
```

Per discussion last year and in the last meeting, the current intention is to get WebKitTestRunner running on iOS Simulator.

Apple only ships stable and beta releases of Mobile Safari, thus if the goal is to have something such that we can show as our "experimental" label, merely running Mobile Safari does not suffice.

Mobile Safari also has no control over viewport size (and this is a feature basically required by testing and nothing else—web developers typically want to test as the browser actually ships to users).